core/src/main/java/io/undertow/UndertowMessages.java

```diff
@@ -619,6 +619,7 @@
     @Message(id = 198, value = "Blocking write timed out after %s nanoseconds.")
     WriteTimeoutException blockingWriteTimedOut(long timeoutNanoseconds);
 
+<<<<<<< HEAD
```

Comment:
You seem to have forgotten to delete this line when resolving the conflict of this PR.

Reply:
@msfm good catch! I'll fix it!